frontend/src/App.constants.ts

```diff
@@ -0,0 +1 @@
+export const RESULT_LIST_MAX_SIZE: number = 20;
```

Comment:
I add a constant to this new constants file. 
We don't want to continue using a 'magic' number or string reference in various parts of the project because if we need to change in one area, we need to change every other area.

Now, we just need to change one thing in the `App.constants` file to change it everywhere it is referenced.

See if there are other places in this code review that can use this 'extract to a single place' principle to avoid repeating these strings and numbers